### PR TITLE
Add parsing task add and delete command functionality

### PR DESCRIPTION
**Pull request by dexter-sim**

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/categoryless/ExitCommand.java [hunk 1]
```diff
@@ -9,7 +9,7 @@
  */
 public class ExitCommand extends Command {
 
-    public static final String COMMAND_WORD = "exit";
+    public static final String COMMAND_WORD = "bye";
 
     public static final String MESSAGE_EXIT_ACKNOWLEDGEMENT = "Exiting Address Book as requested ...";
```
Comment (seetohjinwei) [inline src/main/java/seedu/address/logic/commands/categoryless/ExitCommand.java:14]: Maybe we should change this to "Exiting Task Book" instead?

**Review by seetohjinwei — CHANGES_REQUESTED** [review]
Slight comment change?

File: src/main/java/seedu/address/logic/commands/categoryless/ExitCommand.java (hunk 1 above)
Comment (dexter-sim) [reply src/main/java/seedu/address/logic/commands/categoryless/ExitCommand.java]: Nice catch. I will implement the change.

**Review by seetohjinwei — APPROVED** [review]
lgtm!

**Review by bananamonkey4655 — APPROVED** [review]
LGTM!

**Pull request merged**